Removed watchpoints.dem from all.dem file because it uses libcerf which is not provided in Solaris.
Another change made was to add in the Makefile the parameter "CONFIGURE_OPTIONS  += --without-libcerf" to build without special functions from libcerf.

--- gnuplot-6.0.3/demo/all.dem
+++ gnuplot-6.0.3/demo/all.dem
@@ -573,8 +573,8 @@ load "isosurface.dem"
 # print "********************** file voxel.dem *********************"
 # load "voxel.dem"
 
-print "*********************** file watchpoints.dem *********************"
-load "watchpoints.dem"
+# print "*********************** file watchpoints.dem *********************"
+# load "watchpoints.dem"
 
 pause 2
 print "**********************   all done   ************************"
